game/assets/jak2/text/game_custom_text_en.gs

```diff
@@ -1,16 +0,0 @@
-(group-name "common")
-(language-id 0)
-(text-version jak2)
-
-(#x2000
-  "<PAD_TRIANGLE>"
-  )
-(#x2001
-  "<PAD_CIRCLE>"
-  )
-(#x2002
-  "<PAD_X>"
-  )
-(#x2003
-  "<PAD_SQUARE>"
-  )
```

Comment:
are these already in the new custom text file? otherwise we should copy em over

Reply:
They are in the new format yeah